monai/apps/reconstruction/networks/nets/utils.py

```diff
@@ -0,0 +1,182 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This script contains utility functions for developing new networks/blocks in PyTorch.
+"""
+
+import math
+from typing import Sequence
+
+from torch import Tensor
+from torch.nn import functional as F
+
+
+
+def complex_to_channel_dim(self, x: Tensor) -> Tensor:
+    """
```

Comment:
I was wondering if it is better to name it as `reshape_complex_to_channel_dim` to indicate that this function reshape tensors. What do you think?

Reply:
yeah, sure.